src/themes/creator.js

```diff
@@ -0,0 +1,286 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import type {ThemeT} from '../styles/types';
+import deepMerge from '../utils/deep-merge';
+
+// $FlowFixMe
```

Comment:
What's this for? Hoping we can fix this since Theme is a pretty important thing to flowtype properly.

At the very least maybe we can add a comment to the `$FlowFixMe`